lib/ansible/module_utils/basic.py

```diff
@@ -2685,14 +2685,17 @@ def run_command(self, args, check_rc=False, close_fds=True, executable=None, dat
         prev_dir = os.getcwd()
 
         # make sure we're in the right working directory
-        if cwd and os.path.isdir(cwd):
-            cwd = to_bytes(os.path.abspath(os.path.expanduser(cwd)), errors='surrogate_or_strict')
-            kwargs['cwd'] = cwd
-            try:
-                os.chdir(cwd)
-            except (OSError, IOError) as e:
-                self.fail_json(rc=e.errno, msg="Could not open %s, %s" % (cwd, to_native(e)),
-                               exception=traceback.format_exc())
+        if cwd:
+            if os.path.isdir(cwd):
+                cwd = to_bytes(os.path.abspath(os.path.expanduser(cwd)), errors='surrogate_or_strict')
+                kwargs['cwd'] = cwd
+                try:
+                    os.chdir(cwd)
+                except (OSError, IOError) as e:
+                    self.fail_json(rc=e.errno, msg="Could not open %s, %s" % (cwd, to_native(e)),
```

Comment:
"Could not open" is a bit strange to me, maybe "Could not chdir to" would be better?

Reply:
@mgedmin I'll be happy to change the phrasing to that, but to be fair, I had not looked into it for this PR.

Reply:
```suggestion
                    self.fail_json(rc=e.errno, msg="Could not chdir to %s, %s" % (cwd, to_native(e)),
```